templates/citizen_action/show.html.twig

```diff
@@ -0,0 +1,19 @@
+{% extends 'events/show.html.twig' %}
+
+{% block page_title citizen_action.name %}
+{% block canonical_url url('app_citizen_action_show', { slug: citizen_action.slug }) %}
+
+{% block json_ld %}
+    {% set event = citizen_action %}
```

Comment:
I am not sure. Is it possible to set the `event` variable outside of block ? Like we could avoid the same line 3 times.